### ISSUE TITLE
Following the releasing instructions results in an error when pushing

### ISSUE THREAD

**Issue by mfisher87** [nsidc/earthaccess#833]
"protected branch hook declines"

"Changes must be made through a Pull Request"

Following the instructions as written only works if the releaser bypasses branch protections.

**Comment by jhkennedy** [comment]
Note: There's a relevant discussion of automating the release process in #857

**Comment by asteiker** [comment]
Hi @mfisher87 ! I saw this was mentioned in previously merged PR's. Is this bug still present and needs to be addressed?

**Comment by mfisher87** [comment]
I'm not sure :) It's been some time since I tested. We should try to do our next release exactly following the doc, ideally have someone who's never released before do it without coaching. Then we'll know where our instructions are wrong/insufficient and can update them.

Overhauling the release process feels like a separate thing that we should do either way.

**Comment by asteiker** [comment]
Good call! I know that @chuckwondo recently walked through our release docs for v0.18 and ran into a few issues. I believe you had submitted a small PR to update the release instructions. Do you know if perhaps you addressed this issue, or should this stay open?

**Comment by chuckwondo** [comment]
I believe the instructions should be good now. If there are any rough edges remaining, they are certainly different from the cause of this old issue, so I suggest we close this as fixed, and open a new issue if there are any problems for the next person to attempt a release.
